docs/finality-provider-operation.md

```diff
@@ -928,11 +929,17 @@ This command should ask to
 `confirm transaction before signing and broadcasting [y/N]:` and output the
 transaction hash.
 
-Once the address is set, rewards can be withdrawn by running the
-following command:
+Once the address is set, be sure to set the operational key name in the
+keyring home directory in the `[babylon]` config in `fpd.conf`.
+
+> Important: The `fpd` must be stopped before performing this function.
+
+Once the `fpd` is stopped, run the following to withdraw rewards.
+The finality provider must first be active and have voting power to proceed and
+recieve rewards.
 
 ```shell
-fpd withdraw-reward <type> --from <registered-bbn-address>
+fpd withdraw-reward <type> --from <operational-address>
```

Comment:
IIRC withdraw-reward should be from `registered-bbn-address` and the benefiary is the `operational-address`.